templates/trivy/trivy-sts.yaml

```diff
@@ -0,0 +1,102 @@
+{{- $trivyPVC := .Values.persistence.persistentVolumeClaim.trivy -}}
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
Why it's `StatefulSet`? Isn't trivy a stateless application just like clair?

Reply:
Trivy use file system to save its db and reports.

Reply:
That's correct. As @heww mentioned Trivy adapter is currently stateful as explained here https://github.com/aquasecurity/harbor-scanner-trivy/blob/master/docs/ARCHITECTURE.md

However, we're working on so called [Trivy server - client mode](https://github.com/aquasecurity/trivy#client--server). If we take advantage of that we could go for Deployment instead of StatefulSet.

Reply:
And to be precise Clair is stateful as well. It stores it state in Postgres DB.